### DISCUSSION TITLE
[BUG]及功能建议

### DISCUSSION THREAD

**Discussion by Elletear** [xiaoyaocz/flutter_dmzj#98] [General]
- 两个BUG
- - 浏览记录功能里，不能正常拉取漫画阅读记录，小说可以正常拉取；不知道是不是同步的dmzj云端记录，但当点击阅读记录里小说阅读到新的进度后，只会更新本机记录，这个也许是设计特性？🧐
![comic.png](https://user-images.githubusercontent.com/70796317/225188275-d304ffb9-21e8-47b1-b87c-0cc99c025a57.png)

- - 在阅读小说时，调整阅读方向的会导致当前阅读进度直接跳转到首页，进而丢失阅读进度。

- 功能建议
- - 支持webdav功能同步阅读记录，避免丢失进度。

**Comment by xiaoyaocz** [comment]
#101

**Comment by xiaoyaocz** [comment]
小说阅读云同步在官方APP也不行，应该是DMZJ的问题。
小说阅读进度跳转功能正在开发中。
